src/core/service.c

```diff
@@ -598,6 +598,9 @@ static int service_add_default_dependencies(Service *s) {
         if (!UNIT(s)->default_dependencies)
                 return 0;
 
+        if (UNIT(s)->manager && FLAGS_SET(UNIT(s)->manager->test_run_flags, MANAGER_TEST_RUN_IGNORE_DEPENDENCIES))
+                return 0;
```

Comment:
this has zero effect, no? the calls below check for the flag anyway? just drop it here?

Reply:
yeah this can be dropped now (it was added in the calls below just in the latest push, while this one was there from the beginning)